modules/ROOT/pages/dataweave-troubleshoot.adoc

```diff
@@ -118,6 +118,14 @@ When no MIME type is set on the payload, the MIME type defaults to
 `application/java`, and the content is handled as a `String` instead of a
 JSON object.
 
+=== Reader properties not working inside a Mule application
+
+When working with a transformation inside a Mule application, `input` directives declared inside a script do not work.
+This produces a different behaviour when running the same script in a standalone DataWeave runtime like the one powering the playground.
```

Comment:
```suggestion
Unlike Mule runtime, a standalone DataWeave runtime, such as the runtime in the https://developer.mulesoft.com/learn/dataweave/[DataWeave Playground+], can process a valid MIME type set through the `input` directive in the same DataWeave script.
```

Reply:
@martincousido We need to say what behavior it produces. Does it input the specified MIME type if it is supported by DataWeave? Pls review the suggested change.

Reply:
Basically what happens is that any reader property that you want to specify on your inputs directly in the script, won't work on a mule environment, you have to go and set it in the output of the Message Processor where the data comes from. It's a very generic recommendation because there are multiple cases where this can happen, for example settings a custom separator for a csv file.

Reply:
I re-read it and I think the suggestion is good

Reply:
Only one thing, I think we could remove one runtime.
```suggestion
Unlike Mule runtime, a standalone DataWeave runtime, such as the one in the https://developer.mulesoft.com/learn/dataweave/[DataWeave Playground+], can process a valid MIME type set through the `input` directive in the same DataWeave script.
```